distance_tolerance: 0.3
angle_tolerance: 0.5
k1: 5
k2: 100
pub_topic:
  control: "/dhex/cmd_vel"
sub_topic:
  odom: "/dhex/odom"